tensorflow/lite/tools/cmake/modules/xnnpack/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright 2021 NXP. All Rights Reserved.
```

Comment:
Could you use "Copyright 2022 The TensorFlow Authors. All Rights Reserved."

https://github.com/tensorflow/tensorflow/blob/master/CONTRIBUTING.md